pkgs/top-level/all-packages.nix

```diff
@@ -15710,9 +15685,7 @@ with pkgs;
 
   spirv-llvm-translator = callPackage ../development/compilers/spirv-llvm-translator { };
 
-  sqldeveloper = callPackage ../development/tools/database/sqldeveloper {
-    jdk = oraclejdk;
-  };
+  sqldeveloper = callPackage ../development/tools/database/sqldeveloper { };
```

Comment:
This presumably wants OpenJDK 8, and seemingly with JavaFX going by 294f667121f54f0ba5f77aedb38a0a9fa90bfa0f, which we do not support for OpenJDK 8, since AIUI it was not FOSS then; maybe Zulu or something might ship it? Also, following the `requireFile` instructions, I had to click on “Previous Version” 11 times to reach the download page for the corresponding version from January 2021. I think we ought to ditch this effectively‐unmaintained proprietary Oracle package too.

cc @ardumont who is listed as maintainer and @Ma27 who did the past several updates and who, in a commit message, said “I don't think that we should keep obviously unmaintained software that interacts with database systems.” :)

Reply:
I've gone ahead and simply dropped the package; I'm happy to undo that change if there is consensus to retain the package.

Reply:
Wow, past sins dug up I guess.

I had a use-case for that (Oracle DB for $customer and decent query plan integration for it), but that's not the case since ~2021, i.e. since I don't push updates for this. Given that nobody else seems to care... I'd say kill it with fire.

Reply:
I have to admit I was shocked to see an upstanding citizen like you associated with an Oracle package 😆

Reply:
If you dig deep enough, there's a certain chance you'll find even more dirt (also tooling I needed for said project :see_no_evil:).

Anyways, I guess we can resolve this thread with sqldeveloper being removed.